packages/ui/src/ui/utils/sort-helpers.ts

```diff
@@ -4,6 +4,8 @@ import reduce_ from 'lodash/reduce';
 import some_ from 'lodash/some';
 
 import {OldSortState, SortState} from '../types';
+import {ChaosBundle} from '../store/reducers/chaos_cell_bundles';
+import {TabletBundle} from '../store/reducers/tablet_cell_bundles';
```

Comment:
`utils/sort-helpers.ts` should not contain any references to `TabletBundle`/`ChaosBundle` and other specific entities.